### DISCUSSION TITLE
Add support to retrieve Ui translations of SharePoint sites

### DISCUSSION THREAD

**Discussion by web265p3** [pnp/pnpcore#1312] [Q&A]
In CSOM it was possible to fetch/set values for site columns, menus etc. for different languages.
For example like this:

                    Web web = context.Web;
                    List list = web.Lists.GetByTitle("Documents");
                    context.Load(web);
                    context.Load(list);
                    Field field = list.Fields.GetByTitle("test");
                    context.Load(field, f => f.TitleResource);
                    context.ExecuteQuery();
                    ClientResult<string> result = field.TitleResource.GetValueForUICulture("fr-FR");
                    
                    field.TitleResource.SetValueForUICulture("fr-FR", "测试描述");

I did not find any similar feature in the PnP Core SDK. The "TitleResource" property does not exist ie. Is there any way to archive this? 

I just found a way how to work with multilingual pages in communication sites for pages. But that is not what I need...

**Comment by jansenbe** [comment]
@web265p3 : this is tracked under #1259 and #1292, however I've little bandwidth at the moment to work on this. If someone in the community wants to take a go at this that would be great
